src/components/statistics.js

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Paper } from '@material-ui/core';
+import CountUp from 'react-countup';
+const useStyles = makeStyles((theme) => ({
+    paper: {
+        padding: theme.spacing(1),
+        textAlign: 'center',
+
+    },
+    val: {
+        fontSize: '100px'
+    },
+    name: {
+        fontSize: '30px'
+    },
+    grid: {
+        paddingRight: '100px',
+        paddingLeft: '100px'
+    }
+
+}))
+
+export default function Statistics() {
+
+    const classes = useStyles();
+    return (
+        <Grid container spacing={3} className={classes.grid}>
```

Comment:
Making the grid dynamic would be better i guess!!

Reply:
ok, i will try that